packages/api/endpoints/granules.js

```diff
@@ -148,22 +148,27 @@ async function del(req, res) {
 
   let dynamoGranule;
   let pgGranule;
+
+  // If the granule does not exist in Dynamo, throw an error
   try {
     dynamoGranule = await granuleModelClient.getRecord({ granuleId });
-    pgGranule = await granulePgModel.get(knex, { granule_id: granuleId });
   } catch (error) {
     if (error instanceof RecordDoesNotExist) {
       return res.boom.notFound(error);
     }
     throw error;
   }
 
-  // DEPRECATED this can be removed
-  if (dynamoGranule.detail) {
-    return res.boom.badRequest(dynamoGranule);
+  // If the granule does not exist in PG, just log it
+  try {
+    pgGranule = await granulePgModel.get(knex, { granule_id: granuleId });
+  } catch (error) {
+    if (error instanceof RecordDoesNotExist) {
+      log.info(`Postgres Granule with ID ${granuleId} does not exist`);
```

Comment:
Is this an appropriate way to capture the error? Is there something better I should be using?

Reply:
You also need to do a `throw error` after the `if` block`. Otherwise, you're swallowing all the errors, not just the `RecordDoesNotExist` errors. 

Was it a piece of feedback from Jonathan or I to swallow the error if the PG record doesn't exist? I guess that could make sense since people could be trying to delete a granule before it has been migrated into PG?

Reply:
Yeah, so I believe the idea here is to make the Dynamo record the primary record that determines whether an error should be thrown on DELETE. It's based on this comment thread:

https://github.com/nasa/cumulus/pull/2057#discussion_r574653183

I hear ya on throwing an error if it's not `RecordDoesNotExist` though. I think that makes sense. If we have a PG record that we're trying to get and can't, we should probably bail. Adding that here:

bbf2fb28087ce20890fb6e30709a932b1dd42a8d